oak_runtime/introspection_browser_client/index.tsx

```diff
@@ -16,236 +16,6 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import introspectionEventsProto from './proto/introspection_events_pb';
+import Root from '~/components/Root';
```

Comment:
Is it common practice to use `~` to refer to the root of the hierarchy? I have never seen this.

Reply:
With TS/JS being an organically grown ecosystem, there isn't a clear convention for this. `~` is one of the more common options (as used by [Parcel](https://parceljs.org/module_resolution.html). Other options include `@components/Root` or simply `components/Root`. 

I personally prefer `~` as it does not usually conflict with other paths or NPM packages.